include/cppkafka/utils/backoff_committer.h

```diff
@@ -88,6 +88,18 @@ class BackoffCommitter : public BackoffPerformer {
      * \param consumer The consumer to use for committing offsets
      */
     BackoffCommitter(Consumer& consumer);
+    
+    /**
+     * \brief Constructs an instance using specified values
+     *
+     * \sa BackoffPerformer::BackoffPerformer
+     */
+    BackoffCommitter(Consumer& consumer,
```

Comment:
I don't like doing this at all. The day you want to add one more parameter, you're going to break everyone's code. I think I prefer having something like a `BackoffPerformerConfiguraiton` or something like that where each member is there. If you want to add something new, you'll break the ABI but at least people can just rebuild and that works. As it is, adding a new parameter breaks the API, unless you want to add another constructor but then it becomes unmaintainable.

Reply:
Ok fair enough. I was assuming that class was pretty much closed in terms of config. I'll remove the constructor.

Reply:
I imagine it is but I've had this issue in the past with another library when I thought it was finished but then more arguments kept showing up and it ended up being quite messy.